system-tests/projects/pristine/expected-cypress-js-component-create-react-app-v5/cypress/support/commands.js

```diff
@@ -22,4 +22,4 @@
 //
 //
 // -- This will overwrite an existing command --
-// Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
```

Comment:
Can we just delete these placeholder files? bloats the projects if we don't need them at all. could be a follow up.

Reply:
Yeah we totally could. My priority here was just getting these `system-tests` linted, not to clean out these files. They're not hurting anything so I don't plan on opening a follow-up PR to remove them.